releasing.md

```diff
@@ -0,0 +1,31 @@
+# Releasing UgandaEMR Versions 
+
+UgandaEMR leverages the OpenMRS infrastructure on BinTray offered by JFrog, which is a maven based distribution service. This allows UgandaEMR modules to be used by the greater OpenMRS community, if needed and automation of the processes for release management.
+
+The release process uses the maven release plugin 
+
+## Release Process
+
+The process for releasing is as follows:
+
+1. Setup UgandaEMR CI credentials in the local .m2 repository settings 
```

Comment:
@ssmusoke how is step 1 implemented, also why cant  we use the entire word Continuous Intergration instead of Ci

Reply:
@solemabrothers Which is why we are going to do it together

Reply:
Noted